integration-tests/yarn.lock

```diff
@@ -1,13 +1,29 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
+# This file is generated by running "yarn install" inside your project.
```

Comment:
comment: Why does the case change here? ^^

Reply:
this is just a consequence of running `yarn` in the dir  🤷‍♂️